code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -66,3 +92,42 @@
 	if(toggle && isliving(mover)) //No crawling under a protector
 		return FALSE
 
+/mob/living/simple_animal/hostile/guardian/protector/Destroy()
+	QDEL_LIST_CONTENTS(connected_shields)
+	..()
+
+/obj/effect/guardianshield
+	desc = "A guardian's defencive wall."
+	name = "guardian's shield"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "shield-grey"
+	can_be_hit = TRUE
+	var/mob/living/simple_animal/hostile/guardian/protector/linked_guardian
+	var/left_if_false_right_otherwise = FALSE //Needs a better name, but bools feels better than LEFT and RIGHT being passed
```

Comment:
Could you document this variable a bit better?

Reply:
done